lua/pac3/editor/server/wear.lua

```diff
@@ -108,68 +122,66 @@ duplicator.RegisterEntityModifier("pac_config", function(ply, ent, parts)
 	end)
 end)
 
-function pace.SubmitPart(data, filter)
-	if istable(data.part) then
-		if last_frame == frame_number then
-			table.insert(pace.StreamQueue, {data, filter})
-			pace.dprint("queuing part %q from %s", data.part.self.Name, tostring(data.owner))
-			return "queue"
-		end
+function pace.SubmitPartNow(data, filter)
```

Comment:
This is the new "worker" function. It only has a few changes, which I'll annotate below.